## 50% of end-end tests have passed
```
Total executed tests: 8
Failed tests: 4
Expected to fail tests: 0
Passed tests: 4
```
```
IDs of failed tests: 727230,740490,703133,702843 
```
<h3>Failed tests (4)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/saved-address-name&PR_ID=21448&APK_NAME=21448.apk&TR_CASE_IDS=727230,740490,703133,702843">Rerun failed tests</a></li><br/><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3696089">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `network-dropdown` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
critical/test_wallet.py:249: in test_wallet_balance_mainnet
    self.errors.verify_no_errors()
base_test_case.py:192: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 For the Ether the wrong value 0.0051 is shown, expected 0.0052 in total
E    For the Uniswap the wrong value 0.127 is shown, expected 0.627 in total
E    For the Ether the wrong value 0.0 is shown, expected 0.0001 on Arbitrum
E    For the Uniswap the wrong value 0.0 is shown, expected 0.5 on Arbitrum
```

<br/><br/></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3696084">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` </div><div>Device 2: Wait for element `Button` for max 120s and click when it is available </div></blockquote></p>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:350: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6ead11a6c9ca4281aae1bee6d83a99d1?auth=d4e00cc5915aa699bddc32aafddfd5c8">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/509cebb93ee44d83b369eb8471e145b7?auth=228ca2ef4318b674b313439fbf047dfc">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3696085">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `show-profiles` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:244: in test_restore_multiaccount_with_waku_backup_remove_switch
    self.errors.verify_no_errors()
base_test_case.py:192: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 zQ3...dWXh5 was not restored as a contact from waku backup!
E    zQ3...Vacac was not restored as a contact from waku backup!
E    admin_open was not restored from waku-backup!!
E    member_open was not restored from waku-backup!!
E    admin_closed was not restored from waku-backup!!
E    member_closed was not restored from waku-backup!!
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/960ad556f98441f3befd2146fddd2ad2?auth=297ae350eca2ca7a8b05001f3d56f87d#89">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3696087">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='container']/android.widget.TextView[@text='Ether']/../android.widget.TextView[3]` </div><div>Device 2: `Text` is `0.09019 ETH` </div></blockquote></p>

```
critical/test_wallet.py:189: in test_wallet_send_asset_from_drawer
    self.errors.verify_no_errors()
base_test_case.py:192: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Sender balance is not updated on Etherscan, it is 0.3909 but expected to be 0.391
```

<br/><br/></td></tr></tbody></table></details><h3>Passed tests (4)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3696083">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aab189205c0e445689a473f67991f54b?auth=1b0645f8f84dbc56d05e7b7575e30ed9#96">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/612b4c1ea82c47dda0b6a5b67222208a?auth=6d4a0d8143a914c98797ca43a2bb6385#91">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3696082">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/960ad556f98441f3befd2146fddd2ad2?auth=297ae350eca2ca7a8b05001f3d56f87d#57">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3696086">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3696088">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td></td></tr></tbody></table></details>

--- a/src/status_im/contexts/settings/wallet/saved_addresses/sheets/address_options/view.cljs
+++ b/src/status_im/contexts/settings/wallet/saved_addresses/sheets/address_options/view.cljs
@@ -6,24 +6,21 @@
     [react-native.platform :as platform]
     [status-im.constants :as constants]
     [status-im.contexts.settings.wallet.saved-addresses.sheets.remove-address.view :as remove-address]
-    [status-im.contexts.wallet.common.utils :as utils]
-    [status-im.contexts.wallet.common.utils.networks :as network-utils]
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
 (defn view
-  [{:keys [name full-address chain-short-names address] :as opts}]
-  (let [[_ splitted-address]           (network-utils/split-network-full-address address)
-        open-send-flow                 (rn/use-callback
+  [{:keys [name full-address chain-short-names address customization-color] :as opts}]
+  (let [open-send-flow                 (rn/use-callback
                                         (fn []
                                           (rf/dispatch [:hide-bottom-sheet])
                                           (rf/dispatch [:pop-to-root :shell-stack])
                                           (js/setTimeout #(rf/dispatch [:wallet/select-send-address
                                                                         {:address full-address
                                                                          :recipient
-                                                                         {:label
-                                                                          (utils/get-shortened-address
-                                                                           splitted-address)
+                                                                         {:label name
+                                                                          :customization-color
+                                                                          customization-color
                                                                           :recipient-type :saved-address}
                                                                          :stack-id :wallet-select-address
                                                                          :start-flow? true}])

